includes/class-wc-stripe-intent-controller.php

```diff
@@ -9,6 +9,9 @@
  * Handles in-checkout AJAX calls, related to Payment Intents.
  */
 class WC_Stripe_Intent_Controller {
+
+	const UPE_APPEARANCE_TRANSIENT = 'wc_stripe_upe_appearance';
```

Comment:
Maybe we can use `WC_Stripe_UPE_Payment_Gateway:: UPE_APPEARANCE_TRANSIENT` instead of having the same constant in each file?